helix-view/src/editor.rs

```diff
@@ -704,7 +722,7 @@ pub struct WhitespaceCharacters {
 impl Default for WhitespaceCharacters {
     fn default() -> Self {
         Self {
-            space: '·',    // U+00B7
+            space: '·',   // U+00B7
```

Comment:
`cargo fmt` actually deleted a space here to line it up with the rest of the comments. I don't know if maybe there's some version mismatch causing issues

Reply:
Yeah this happens for me all the time with a nightly rust version, now it got even "worse" since finally let-else is formatted in nightly (maybe even in the latest stable, I don't know). I wonder whether it makes sense to use a (stable) up-to-date rustfmt while otherwise keeping the MSRV version, though this may unnecessarily complexify stuff.

Reply:
you are using rustfmt installed via rustup it should automatically use the right rustfmt version (the one from rust 1.65) already I think. Its pinned to 1.65 in the `rust-toolchain.toml` file. That's the version that will be used by CI too and should be used during development. We could pin a newer version but I don't think that's really worth the effort

Reply:
Yeah I agree, though it may make sense to reformat the code (without this particular change we are commenting on) with the new let-else formatting feature, since the MSRV version shouldn't care anyway as it doesn't format let-else code.

Reply:
yeah let..else in particular would be reasonable, basically our policy has been: we accept PRs with fixes for clippy lints/fmt changes from nightly as long as they don't break stable CI